src/components/ContentHighlights/data/constants.js

```diff
@@ -306,3 +330,68 @@ export const TEST_COURSE_HIGHLIGHTS_DATA = [
     ],
   },
 ];
+
+export const testCourseData = [
```

Comment:
[clarification] Is this only used by tests?

Reply:
Yep, it reflects sample data coming from the algolia's search clients `hits` that have been destructured to Content cards metadata